jbpm-installer/src/main/resources/db/ddl-scripts/sybase/sybase-jbpm-schema.sql

```diff
@@ -1,19 +1,19 @@
     create table Attachment (
-        id numeric(19,0) identity not null,
+        id bigint identity not null,
```

Comment:
No, we don't. But AFAIK we use SybaseASE15Dialect in our tests currently.

Reply:
Issue happens when validating with Hibernate using its SybaseASE157Dialect (which extends from SybaseASE15Dialect): according to this dialect "bigint" is registered for type code BIGINT, instead of "numeric(19,0)".
We can implement our custom validator overwriting SybaseASE157Dialect (or use SybaseDialect) but from my point of view is like "changing the validation rules for keeping the script".
@sutaakar @MarianMacik wdyt?

Reply:
Finally, used a custom dialect and a new resolver also for jBPM. Notice that now validation is against the standard dialects (latest) for each database, because I didn't find a way to define a validator if already exists hibernate.dialect property (this property has to be commented).

Reply:
So why was this file omitted?

Reply:
I guess hibernate.dialect property coming from Sybase dballocator was previously other dialect (probably H2Dialect). Since the test environment was not prepared to receive Sybase dialects, as it has had to be added a new DatabaseType (see method _getDatabaseTypeBySQLDialect_) for considering these scripts.